homeassistant/components/android_ip_webcam/sensor.py

```diff
@@ -138,7 +138,7 @@ async def async_setup_entry(
         + ["audio_connections", "video_connections"]
     ]
     async_add_entities(
-        [IPWebcamSensor(coordinator, description) for description in sensor_types]
+        [IPWebcamSensor(coordinator, description) for description in sensor_types], True
```

Comment:
The coordinator should be updated before we set up the platforms. We don't need to refresh again here.